app/controllers/AuthorizedToViewController.scala

```diff
@@ -50,8 +50,8 @@ class AuthorizedToViewController @Inject()(authenticate: IdentifierAction,
                                            authorisedToViewSearch: authorised_to_view_search,
                                            authorisedToViewSearchResult: authorised_to_view_search_result,
                                            authorisedToViewSearchNoResult: authorised_to_view_search_no_result,
-                                           eoriNumberFormProvider: EoriNumberFormProvider)(
```

Comment:
on line 40 of this file we have a scalastyle off comment, is this still required?

Reply:
yes, its is required. It needs to be refactored to reduce the cyclomatic complexity. Should be done as part of separate ticket